openquake/risklib/reinsurance.py

```diff
@@ -149,7 +192,7 @@ def parse(fname, policy_idx):
     rmodel = nrml.read(fname).reinsuranceModel
     fieldmap = {}
     fmap = {}  # ex: {'deductible': 'Deductible', 'liability': 'Limit'}
-    treaty = dict(id=[], type=[], deductible=[], limit=[])
+    treaty = dict(id=[], lineno=[], type=[], deductible=[], limit=[])
```

Comment:
Not so happy the `lineno` being promoted to be a real field, stored in the datastore, while it is just an auxiliary field not needed by the engine.

Reply:
Done